common-lib/src/cmr/common/util.clj

```diff
@@ -616,10 +616,14 @@
 
 (defn map-matches-path-values?
   "Returns true if the map matches the given path values. Path values are
-  described in the map->path-values function documentation."
+  described in the map->path-values function documentation.  When a value is sequential,
+  any map which matches at least one of the contained values will be considered to match
+  (i.e. the values are ORed)"
   [path-values m]
   (every? (fn [[path value]]
-            (= (get-in m path) value))
+            (or
```

Comment:
flatten wont work as it will mess up deeply nested structures, which may not be realistic, but exist in the unit test data, e.g.:
```
                   {:a {:b "B"
                        :c "C"}
                    :b {"d" "D"}
                    :c [{:e "E":f "F"} [{:g "G":h "H"}]]}
```
working on some in between solution